dev-lang/spidermonkey/spidermonkey-115.10.0.ebuild

```diff
@@ -78,7 +78,7 @@ BDEPEND="${PYTHON_DEPS}
 			clang? (
 				sys-devel/lld:18
 				sys-devel/clang:18
-				virtual/rust:0/llvm-18
+				<virtual/rust-1.78:0/llvm-18
```

Comment:
I don't think anything will satisfy this for 1.78 in the 0/llvm-18 subslot but I don't think it's _wrong_ either.